partiql-ast/src/main/java/org/partiql/ast/AstVisitor.java

```diff
@@ -61,6 +61,58 @@ public R visitStatement(Statement node, C ctx) {
         return node.accept(this, ctx);
     }
 
+    //
+    // DDL
+    //
+    public R visitDdl(Ddl node, C ctx) {
+        return node.accept(this, ctx);
```

Comment:
nit: could just use the default visit

```suggestion
        return defaultVisit(node, ctx);
```